.github/workflows/release.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       - name: Run Package build
         run: |
           constraintVersion="${{ github.ref_name }}"
-          ytt -f config/release > ./tmp/output-config.yml
+          ytt -f config/release > ./config/relase-config.yml
```

Comment:
release spelling is misspelled as relase

Reply:
Refactored.